relayer/workers/worker.go

```diff
@@ -121,6 +124,6 @@ func (wp WorkerPool) Run() error {
 	return eg.Wait()
 }
 
-func (wp WorkerPool) getLogger() *logrus.Entry {
+func (wp WorkerPool) getDefaultLogger() *logrus.Entry {
```

Comment:
`defaultLogger` is a more idiomatic getter name in golang. No need to prepend `get`.